hayeon/baekjoon/브루트포스/2798-블랙잭.js

```diff
@@ -0,0 +1,39 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const [count, max] = input[0].split(" ").map(Number);
+  const cards = input[1].split(" ").map(Number);
+  let result = 0;
+
+  for (let i = 0; i < count - 2; i++) {
+    for (let j = i + 1; j < count - 1; j++) {
+      for (let k = j + 1; k < count; k++) {
+        const sum = cards[i] + cards[j] + cards[k];
+
+        if (sum <= max && sum > result) {
+          result = sum;
+        }
+      }
+    }
+  }
+  console.log(result);
+});
```

Comment:
```suggestion
  const [count, max] = input[0].split(" ").map(Number);
  const cards = input[1].split(" ").map(Number);
  let result = 0;

  for (let i = 0; i < count - 2; i++) {
    for (let j = i + 1; j < count - 1; j++) {
      for (let k = j + 1; k < count; k++) {
        const sum = cards[i] + cards[j] + cards[k];

        if (sum <= max) {
          result = Math.max(result, sum);
        }
      }
    }
  }
  console.log(result);
});
```
- 풀이 잘봤습니다 🙇‍♂️
- 아래와 같이 Math.max를 이용해 값을 갱신하는 방법도 존재합니다 ! 
```js
if (sum <= max) {
  result = Math.max(result, sum);
}
```

Reply:
오오 처음에 저부분을 안넣어서 헤맸는데 Math.max로도 이용할 수 있군여.... 뭔가 더 이해하기 쉬운 코드인거같아요!!!! 감사합니다!!👍🏻